src/components/ReplyComment/index.js

```diff
@@ -129,7 +135,7 @@ const ReplyCommentId = ({
                           level={parseInt(level, 10) + 1}
                           updateVote={updateVote}
                           onLongPress={() => {
-                            showAlertDelete(itemReply, false, () => getThisComment(true));
+                            showAlertDelete(itemReply, false, updateComment);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 122, the `showAlertDelete` function now takes `updateComment` as an argument instead of `() => getThisComment(true)`. Ensure that `showAlertDelete` can handle this change and that it calls `updateComment` correctly.

```diff
-                             showAlertDelete(itemReply, false, updateComment);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->